docs/admin/configuration.md

```diff
@@ -15,6 +15,7 @@ The following table shows the environment variables available for Perseo configu
 | PERSEO_ENDPOINT_HOST               | Host where the CEP will listen.                                                                                                                                                                                                                                               |
 | PERSEO_ENDPOINT_PORT               | Port where the CEP will listen.                                                                                                                                                                                                                                               |
 | PERSEO_MONGO_ENDPOINT              | Endpoint (`host[:port]`) list for Mongo DB.                                                                                                                                                                                                                                   |
+| PERSEO_MONGO_DB_NAME               | Mongo Data Base name .                                                                                                                                                                                                                                                        |
```

Comment:
Does it has equivalence y config.js setting?

Reply:
Equivalent is mongo.url:
https://github.com/telefonicaid/perseo-fe/blob/2767b2fab8c6378dbe5fe0e86a23d115faa6e235/config.js#L63-L68

Reply:
Maybe all the PERSEO_MONGO_XXXX should be unified in a single PERSEO_MONGO_URI (similar to what we did in Orion 4.0.0, see https://github.com/telefonicaid/fiware-orion/releases/tag/4.0.0).

Not for this PR anyway... NTC

Reply:
```suggestion
| PERSEO_MONGO_DB_NAME               | Mongo Data Base name (default value is `cep`).                                                                                                                                                                                                                                                        |
```